app.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const bodyParser = require('body-parser')
+const cookieSession = require('cookie-session')
+const bcrypt = require('bcrypt')
+
+const db = require('./db')
+
+const app = express();
+
+app.use(bodyParser.urlencoded({ extended: false }))
+
+app.use(cookieSession({
+  name: 'session',
+  secret: 'reallyLongComplicatedString'
+}))
+
+app.set('view engine', 'pug')
+
+app.get('/', (request, response) => {
+  const user = request.session.email
+  response.render('index', { user })
+})
+
+app.get('/signup', (request, response) => {
+  if (request.session.email) {
+    response.redirect('/')
+  } else {
+    response.render('signup')
+  }
+})
+
+app.post('/signup', (request, response) => {
+  const email = request.body.email
+  const password = request.body.password
+  const passwordConfirm = request.body['password-confirmation']
+  let errorMessage = undefined
```

Comment:
`let errorMessage = undefined` has the same effect as just saying `let errorMessage`